builds/e2e/connectivity.yaml

```diff
@@ -111,9 +120,9 @@ jobs:
         displayName: 'Download Edgelet Artifacts'
         inputs:
           buildType: specific
-          project: $(vsts.project)
```

Comment:
The definition of `vsts.project` can be removed up at the top of this YAML file.

Reply:
Fixed.